code-challenges.js

```diff
@@ -1,51 +1,45 @@
 // ASSESSMENT 4: JavaScript Coding Practical Questions
-
 // --------------------1) Create a function that takes in an array. Each time the function is run, remove the first item from the array and shuffle the remaining content. If the array is empty, return "The array is empty."
-
 var collections = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example1 (can be a different order): ["yellow", "blue", "pink", "green"]
 // Expected output example2 (can be a different order): ["blue", "green", "pink"]
 // Expected output example3 (can be a different order): ["pink", "green"]
-
-
-
-
-
+//
+const minus1 = (array) => {
+  var collections = array.shift(0)
+  for(let i = 0; i < 0; i++){
+    return array
+  }
+}
+console.log(minus1(collections))
```

Comment:
This only returns back 'undefined'. I can't seem to get it to work on my end here. What does it output for you?